packages/mds-schema-validators/package.json

```diff
@@ -16,12 +16,14 @@
   ],
   "scripts": {
     "build": "tsc --build tsconfig.build.json",
-    "test": "DOTENV_CONFIG_PATH=../../.env nyc --use-spawn-wrap --lines 75  ts-mocha --project ../../tsconfig.json"
+    "test": "DOTENV_CONFIG_PATH=../../.env jest"
```

Comment:
Do the old tests still run after this change?

Reply:
Yes.